screens/Movies/Movies.tsx

```diff
@@ -0,0 +1,81 @@
+import {useQuery} from '@tanstack/react-query';
+import React, {useMemo} from 'react';
+import {View, Text} from 'react-native';
+import useMoviesStore from '../../stores/moviesStore';
+import {getFavoritedMovies, getMovies} from '../../utils/api';
+import {FavoritedMovieId, Movie} from '../../utils/interfaces';
+import {globalStyles} from '../../globalStyles';
+import MoviesList from '../../components/MoviesList/MoviesList';
+import MovieCard from '../../components/MovieCard/MovieCard';
+import SkeletonList from '../../components/SkeletonList/SkeletonList';
+import useFavoritedMoviesStore from '../../stores/favoritedMoviesStore';
+
+export default function Movies() {
+  const movies = useMoviesStore(state => state.movies);
```

Comment:
why not
const {  movies, search } = useMoviesStore(state=>({movies : state.movie, search: state.move}))